args.js

```diff
@@ -23,13 +23,14 @@ const DEFAULT_ARGUMENTS = {
 
 module.exports = function parseArgs (args) {
   dotenv.config()
+  const TRUST_PROXY = process.env.TRUST_PROXY || ''
   const commandLineArguments = argv(args, {
     configuration: {
       'populate--': true
     },
     number: ['port', 'inspect-port', 'body-limit', 'plugin-timeout', 'close-grace-delay'],
     string: ['log-level', 'address', 'socket', 'prefix', 'ignore-watch', 'logging-module', 'debug-host', 'lang', 'require', 'config', 'method'],
-    boolean: ['pretty-logs', 'options', 'watch', 'verbose-watch', 'debug', 'standardlint', 'common-prefix', 'include-hooks'],
+    boolean: ['pretty-logs', 'options', 'trustProxy', 'watch', 'verbose-watch', 'debug', 'standardlint', 'common-prefix', 'include-hooks'],
```

Comment:
it has multiple shapes.

fwis, string and boolean are the most used so I would support it or I would rename it to `FASTIFY_ENABLE_TRUST_PROXY`